### PR TITLE
create my profile

### PR DESCRIPTION
**Pull request by H4ruk1**

### CODE REVIEW

**Review by barrycumbie — CHANGES_REQUESTED** [review]
just missing your github username in text of link

**Comment by barrycumbie** [comment]
@H4ruk1 don't forget to give me the link to your ```active test site``` and the Nu link to your ```active test site```

for example, look at #20

File: docs/dev-profile/Haruki-dev.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <h3>Skills</h3>
+        <p>Python, Java, Javascript, PHP, HTML, CSS, SQL</p>
+        <h3>Topic discussed in class</h3>
+        <ul>
+          <li>Github: a Version Control Platform.</li>
+          <li>Git Branch</li>
+          <li>Terminal command review</li>
+        </ul>
+        <br>
+     /body>
```
Comment (barrycumbie) [inline docs/dev-profile/Haruki-dev.html:38]: missing < on linke 38

File: docs/dev-profile/Haruki-dev.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		64-bitoperating system, x64-based processor
+        </p>
+        <h3>Skills</h3>
+        <p>Python, Java, Javascript, PHP, HTML, CSS, SQL</p>
+        <h3>Topic discussed in class</h3>
+        <ul>
+          <li>Github: a Version Control Platform.</li>
+          <li>Git Branch</li>
+          <li>Terminal command review</li>
+        </ul>
```
Comment (barrycumbie) [inline docs/dev-profile/Haruki-dev.html:36]: don't for get the link to the Nu validation checker.

**Review by barrycumbie — CHANGES_REQUESTED** [review]

**Comment by H4ruk1** [comment]
@barrycumbie , I just updated index.html file for dev-02.

**Comment by barrycumbie** [comment]
@H4ruk1 There are 3 links that I need for this assignment, 2 of them in the comments here and 1 in the code 

As a comment: 
1. A link to your active test site: 
2. A link to the nu checker of your active test site (note: this is the one you currently have in your code): https://validator.w3.org/nu/?doc=https%3A%2F%2Fh4ruk1.github.io%2Fdev-challenge-02%2Fdocs%2Fdev-profile%2FHaruki-dev.html

In the code 
3. A link to the nu checker of your profile ONCE IT BECOMES MERGED into our repo. So, you are creating a link that DOES NOT WORK yet BUT WILL WORK after it is merged. 
> hint: it will look like this:
> https://validator.w3.org/nu/?doc=https%3A%2F%2Fh4ruk1.github.io%2Fdev-challenge-02%2Fdocs%2Fdev-profile%2FHaruki-dev.html
> BUT it will not be your username, it will be mine AND my repo doesn't need the ```docs/``` in the URL.

**Comment by barrycumbie** [comment]
Hey @barrycumbie here is my 

active test site: ...link

nu checker: https://validator.w3.org/nu/?doc=https%3A%2F%2Fh4ruk1.github.io%2Fdev-challenge-02%2Fdocs%2Fdev-profile%2FHaruki-dev.html

**Comment by H4ruk1** [comment]
Hey @barrycumbie here is my

active test site: https://h4ruk1.github.io/dev-challenge-02/docs/dev-profile/Haruki-dev.html

nu checker: https://validator.w3.org/nu/?doc=https%3A%2F%2Fh4ruk1.github.io%2Fdev-challenge-02%2Fdocs%2Fdev-profile%2FHaruki-dev.html

**Comment by barrycumbie** [comment]
@H4ruk1 thank you for the 2 links! You have one more small issue to correct and then you will be done. 

In your code on line 56

```html
   <a href="https://validator.w3.org/nu/?doc=https%3A%2F%2Fh4ruk1.github.io%2Fdev-challenge-02%2Fdocs%2Fdev-profile%2FHaruki-dev.html">Nu validation link</a>
```

you are missing two things. 

1. specify the target, like you did for your GitHub Profile link in line 30

```html
<a href="https://github.com/H4ruk1" target="_blank">H4ruk1 </a>
```

you need to add ```target="_blank"``` to line 56. 

2. A link to the nu checker of your profile ONCE IT BECOMES MERGED into our repo. 
3. 
4. So, you are creating a link that DOES NOT WORK yet BUT WILL WORK after it is merged.

hint: it will look like this:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fh4ruk1.github.io%2Fdev-challenge-02%2Fdocs%2Fdev-profile%2FHaruki-dev.html
BUT it will not be your username, it will be mine AND my repo doesn't need the docs/ in the URL.

**Comment by barrycumbie** [comment]
hey @H4ruk1 good job...now you can submit this SAME pr for the Dev-02

**Comment by barrycumbie** [comment]
Congrats, welcome to our REPO as a COLLABORATOR 

image

Don't forget to ACCEPT my invitation via email (check your SPAM if you don't see it)

**Pull request merged**